publish.yml

```diff
@@ -62,6 +62,6 @@ extends:
           tag: beta
           publishRequiresApproval: false
 
-        apiScanExcludes: 'package/third_party/conpty/1.22.250204002/win10-arm64/*.*'
+        apiScanExcludes: 'package/third_party/conpty/1.22.250314001/win10-arm64/*.*'
```

Comment:
@rzhao271 should this just be `package/third_party/conpty/*` or `package/third_party/conpty/*/win10-arm64/*.*`?